spec/index.bs

```diff
@@ -5884,10 +5884,32 @@ details.
                 type, and
                 [=interpolation=]
                 of the input.
-            - There is less than |device|.limits.{{supported limits/maxInterStageShaderComponents}}
-                components of user-defined outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
-            - There is less than |device|.limits.{{supported limits/maxInterStageShaderComponents}}
-                components of user-defined inputs for |descriptor|.{{GPURenderPipelineDescriptor/fragment}}.
+            - [$validating inter-stage interfaces$](|device|, |descriptor|) succeeds.
+</div>
+
+<div algorithm>
+    <dfn abstract-op>validating inter-stage interfaces</dfn>(device, descriptor)
+
+    **Arguments:**
+        - {{GPUDevice}} |device|
+        - {{GPURenderPipelineDescriptor}} |descriptor|
```

Comment:
formatting nit (these parse as separate paragraphs at the same level, not nested):

```suggestion
    **Arguments:**

    - {{GPUDevice}} |device|
    - {{GPURenderPipelineDescriptor}} |descriptor|
```